stable/pipelines/templates/mysql/configmap-kf.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 data:
 
   health_check.sh: |
-    mysqladmin status
+    mysql -u root -S /var/run/mysqld/mysql.sock -e 'SELECT 1'
```

Comment:
Don't change this line. We changed it for MySQL 8. As you are still using 5.6 the previous command should be ok